backend/schemas/blockContent.ts

```diff
@@ -0,0 +1,67 @@
+import {defineType, defineArrayMember} from 'sanity'
```

Comment:
this one is probably unused

Reply:
It is used.

If I comment it CMS is down